src/util/FetchUtil.ts

```diff
@@ -24,33 +42,37 @@ export async function fetchDataset(response: Response, converter: Representation
 Promise<Representation>;
 export async function fetchDataset(input: string | Response, converter: RepresentationConverter, body?: string):
 Promise<Representation> {
-  let response: Response;
   if (typeof input === 'string') {
-    response = await fetch(input);
-  } else {
-    response = input;
+    // Try content negotiation to parse quads from uri
+    const quadArray = await unrollQuads((await rdfDereferencer.dereference(input)).quads);
+    // Make Representation object
+    const representation = new BasicRepresentation(quadArray, INTERNAL_QUADS);
```

Comment:
```suggestion
    const representation = new BasicRepresentation(quadArray, input, INTERNAL_QUADS, false);
```
Not super relevant but it creates a more correct representation.

Reply:
Will do that, but input will have to be wrapped as follows:
``` ts
const identifier: ResourceIdentifier = { path: url };
const representation = new BasicRepresentation(quadArray, identifier, INTERNAL_QUADS, false);
```

Reply:
Ah I thought the constructor also accepted strings. Would still put it on a single line though 
```ts
const representation = new BasicRepresentation(quadArray, { path: input }, INTERNAL_QUADS, false);
```
Unless this makes the line longer than 120 chars since as I said this is mostly irrelevant and mostly for style points 😄.